src/summary/SummaryMetadata.jsx

```diff
@@ -1862,8 +1865,15 @@ export default class SummaryMetadata {
             }  
         });
         
-        return [aliveSeries, deceasedSeries];
+        return [aliveSeries, deceasedSeries];*/
+       try {

```

Comment:
[DL3]  Fix spacing in try catch, should be indented 4 spaces.
[DL4]  Remove commented code in `getTreatmentData` unless we want this for a specific reason.